samples/Client/DaprClient/Readme.md

```diff
@@ -22,3 +22,92 @@ The client sample shows how to make Dapr calls to publish events, save state, ge
   Got State: my data
   Deleted State!
  ```
+
+## Invoking Services
+This solution contains a sample REST *RoutingSample* service, which implements a simple banking application in ASP.NET core.
+The service provides following operations:
+- balance
+- withdraw
+- deposite
+
+The service is a typical HTTP/S REST service. However, by using **dapr**, another application can invoke an operation on this service by using not only REST.
+Following sample demonstrates how to use the gRPC with Dapr .NET SDK to invoke REST service operations.
+
+Operation *balance* is a GET-operation mapped as (see startup.cs in the *RoutingService*):
+ ```c#
+ endpoints.MapGet("{id}", Balance);
+ ```
+
+It can be invoked via HTTTP/S by using following url:
+ ```
+curl -X GET http://127.0.0.1:5000/17
+ ```
+
+If you are developing the .NET client and want to invoke this operation via HTTP/S, you can also use *HttpClient* API.
+The **dapr** supports out-of-the-box invokation of the service operation via gRPC.
+To demonstrate this, take a look on the method *InvokeBalanceServiceOperationAsync*.
+ ```c#
+        internal static async Task InvokeBalanceServiceOperationAsync(DaprClient client)
+        {
+           // Add the verb to metadata if the method is other than a POST
+            var metaData = new Dictionary<string, string>();
+            metaData.Add("http.verb", "GET");
+
+            // Invokes a GET method named "hello" that takes input of type "MyData" and returns a string.
+            var res = await client.InvokeMethodAsync<object>("routing", "17", metaData);
+        }   
+ ```
+
+
+
+Second operation *withdraw* can be invoked by HTTP/S POST request, but also triggered as a *cloud event* by publishing the event to the topic with name 'withdraw'.
+To enable this, the operation is mapped as:
+ ```c#
+endpoints.MapPost("withdraw", Withdraw).WithTopic("withdraw");
+ ```
+and it can also be invoked (triggered) by following url:
+
+ ``` 
+curl -X POST http://127.0.0.1:5000/withdraw -H "Content-Type: application/json" -d "{ \"id\": \"17\", \"amount\": 1 }"
+ ```
+
+
+The method *InvokeWithdrawServiceOperationAsync* demonstrates how to use DAPR .NET SDK to invoke a REST/POST operation via gRPC.
+ ```c#
```

Comment:
Lets not put the code snippets in document,they become hard to manage when code changes.

Reply:
Not sure what do you mean. The reason for this documentation chapter is to demonstrate how to invoke services. Right now, it very hard to do anything without examples. 
Are you suggesting to remove code from *readme* or you suggest some changes?

Reply:
Yes, I am suggesting to remove code snippet from readme.md. When api changes, the doc can go out of sync as they don't generate compile time errors.

Reply:
I have simplified samples in the last commit.